src/data/services/DiscoveryDataApiService.js

```diff
@@ -12,7 +13,7 @@ class DiscoveryDataApiService {
       exclude_utm: 1,
     };
     const url = `${discoveryBaseUrl}/courses/${uuid}/`;
-    return apiClient.get(url, {
+    return getAuthenticatedHttpClient().get(url, {
```

Comment:
Should this file just instantiate this once at the top rather than per call? Or is there a potential token issue or something that way?

Reply:
It does some kind of async lookup - such that I think it's good to not do it globally anymore? Like, it's only defined after initialization time? And I don't know when globals get defined. I saw another frontend do this pattern, so I just stole it.